docs/development/cores/core-options-translation.md

```diff
@@ -0,0 +1,113 @@
+# Adding Translations
+
+## Setting up translation with Crowdin
+
+The scripts and workflows referenced here, as well as some sample `core_options` files, can be found in the [RetroArch translation example](https://github.com/libretro/RetroArch/tree/master/libretro-common/samples/core_options/example_translation).
+
+### Requirements 
+
+Make sure the core is libretro conformant:
+both `libretro_core_options.h`, containing the English texts, and
+`libretro_core_options_intl.h`, containing all already existing
+translations, if any, must be present in the same directory.
+
+> Please note: `libretro_core_options_intl.h` does not need to contain
+anything, if no translations exist or none of them should be preserved.
+
+The scripts are not compatible with text filled in by macros or during run time.
+The procedure should not fail - but those texts will not be made translatable.
+
+Also, please verify the existence and correct use of
+
+   `#ifdef HAVE_LANGEXTRA`
+   
+and/or
+   
+   `#ifndef NAVE_NO_LANGEXTRA`
+   
+pre-compiler instructions in `libretro_core_options.h` to remove any
+references to additional languages on platforms which cannot handle them,
+e.g. due to limited RAM.
+For an example, refer to an up-to-date core, like [gambatte-libretro](https://github.com/libretro/gambatte-libretro/blob/master/libgambatte/libretro/libretro_core_options.h).
+
+> Make sure `options_intl` in `libretro_core_options.h` correctly references the `_intl` options, or the translations will not be applied!
+
+### Adding automatic Crowdin sync
+
+Place the `intl` and `.github` folders from the [sample folder](https://github.com/libretro/RetroArch/tree/master/libretro-common/samples/core_options/example_translation), including content, into the root
+of the repository.
+
+In `.github/workflows` are two files: `crowdin_prep.yml` & `crowdin_translate.yml`.
+
+In each of those are placeholders, which need to be replaced.
+
+For convenience, one can run `intl/activate.py`, which will try to find
+the `libretro_core_options.h` file as well as identify the core name to
+fill those placeholders with.
+
+Even then, one should still check if it produced the correct result:
+
+For `crowdin_prep.yml`:
+> **NOTE:** Please verify, that this workflow watches the correct branch!
```

Comment:
Including example values "master or main" might make this clearer.

Reply:
Actually, it's pretty obvious in the YAML, so fine as-is.

Reply:
Well, I added an example anyway. Thank you.